test/docker-compose.yml

```diff
@@ -12,13 +12,14 @@ services:
       MYSQL_PASSWORD: meroxapass
       MYSQL_ROOT_PASSWORD: meroxaadmin
     ports:
-      - "3306:3306"
+      - '3306:3306'
     networks:
       - main
     healthcheck:
-      test: ["CMD", "mysqladmin", "ping", "-h", "localhost"]
+      test: ['CMD', 'mysqladmin', 'ping', '-h', 'localhost']
+      interval: 1s
       timeout: 20s
-      retries: 10
+      retries: 30
```

Comment:
Hmm, is it really needed?

Reply:
It cuts in more than half startup times of my local mysql docker service, so I believe it is worth it. By [default](https://docs.docker.com/reference/compose-file/services/#healthcheck:~:text=has%20the%20same%20default%20values) it has [30s for the interval value](https://docs.docker.com/reference/dockerfile/#healthcheck).

I got to admit that I just tinker around the healthcheck for each connector until I find something that starts fast enough and doesn't break ci. It also makes test times much faster on ci, which is very helpful when debugging some strange bugs there.